apps/cli/src/export/word/schema.ts

```diff
@@ -152,20 +152,19 @@ export function getNodesAndMarks(styles?: Styles) {
     cite(state, node) {
       const { kind, key, text } = node.attrs as Nodes.Cite.Attrs;
       const oxa = oxaLinkToId(key);
-      if (!oxa) return;
       // TODO: This is probably very wrong
-      const id = oxa.id || oxa.block.block.slice(0, 8);
+      const id = oxa?.id || oxa?.block.block.slice(0, 8);
       switch (kind) {
         case ReferenceKind.cite:
           state.text(text);
           break;
         case ReferenceKind.table:
         case ReferenceKind.code:
         case ReferenceKind.fig:
```

Comment:
I don't think fig/eqn references are going to work quite yet since they are using `oxa` ids still.

Reply:
Ok - cross references don't work, but they just need support in schema: 
![image](https://user-images.githubusercontent.com/9453731/181854240-e8e2297e-7d88-4557-aae9-2b349a247853.png)

Other oxa id stuff is also not going to work, but that probably won't come up much in local md files.